### ISSUE TITLE
isort 5.5.4 introduces syntax error by removing closing paren

### ISSUE THREAD

**Issue by maxdjohnson** [PyCQA/isort#1539]
Hello,

I've found what I believe to be a bug in isort. This is the minimum case I can find that reproduces the issue:

```
"""Foobar
    from {}""".format(
    "bar",
)
```

Saving this to a file `t.py` and running isort on it does the following for me

```
> isort t.py
Fixing /Users/mjohnson/t.py
> cat t.py
"""Foobar
    from {}""".format(
    "bar",
```

Note the missing closing paren from the last line, which is a syntax error. I have tested this on a fresh virtualenv with only `isort==5.5.4` with both python 3.6 and 3.8.

**Comment by timothycrosley** [comment]
Thank you for reporting! this has been fixed in isort 5.5.5+ with several additional regression tests added.

~Timothy

**Comment by maxdjohnson** [comment]
Appreciate the quick fix!
